selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/factory/CheckOpenFileFeatureTest.java

```diff
@@ -73,7 +74,7 @@ public void tearDown() throws Exception {
     factoryServiceClient.deleteFactory(FACTORY_NAME);
   }
 
-  @Test
+  @Test(groups = FLAKY)
```

Comment:
There is only one test in the test class. So, we can mark all class as FLAKY to exclude whole class from test execution.